extern/bullet2/src/BulletDynamics/Vehicle/btRaycastVehicle.h

```diff
@@ -61,7 +61,7 @@ class btRaycastVehicle : public btActionInterface
 	btVehicleRaycaster*	m_vehicleRaycaster;
 	btScalar		m_pitchControl;
 	btScalar	m_steeringValue; 
-	btScalar m_currentVehicleSpeedKmHour;
+	btScalar m_currentVehicleSpeedMetersPerSec;
```

Comment:
Hi! I don't know if this is a good idea to modify bullet sources... I'd use the existing m_currentVehicleSpeedKmHour, getCurrentSpeedKmHour() const